lib/service/service.go

```diff
@@ -303,13 +303,29 @@ func (process *TeleportProcess) initAuthService(authority auth.Authority) error
 	}
 
 	// create the audit log, which will be consuming (and recording) all events
-	// and record sessions
+	// and recording all sessions.
 	var auditLog events.IAuditLog
 	if cfg.Auth.NoAudit {
+		// this is for teleconsole
 		auditLog = &events.DiscardAuditLog{}
-		log.Warn("the audit and session recording are turned off")
+
+		warningMessage := "Warning: Teleport audit and session recording have been " +
+			"turned off. This is dangerous, you will not be able to view audit events " +
+			"or save and playback recorded sessions."
+		log.Warn(warningMessage)
 	} else {
-		auditLog, err = events.NewAuditLog(filepath.Join(cfg.DataDir, "log"))
+		// check if session recording has been disabled. note, we will continue
+		// logging audit events, we just won't record sessions.
+		recordSessions := true
+		if cfg.Auth.ClusterConfig.GetSessionRecording() == services.RecordOff {
+			recordSessions = false
+
+			warningMessage := "Warning: Teleport session recording have been turned off. " +
```

Comment:
same message duplicated twice, make a constant?

Reply:
It's a different message. In the other message, the audit log and session log were disabled.